document/FileFormat.mdk

```diff
@@ -910,19 +938,25 @@ An `invocation` object contains information describing the invocation of the ana
 ### `commandLine` property { #invocation-commandLine }
 
 An `invocation` object may contain a property named `commandLine` whose value is a string
-containing the runtime parameters with which the tool was invoked.
-For command line tools, this string may consist of the completely specified command line used to invoke the tool.
+containing the completely specified command line used to invoke the tool,
+starting with the path to the tool's executable or script file.
```

Comment:
Good catch. Changed to: "... starting with the name of the tool's executable or script file, optionally qualified by the relative or absolute path to the file."

---
In reply to: [75796568](https://github.com/sarif-standard/sarif-spec/pull/256#discussion_r75796568) [](ancestors = 75796568)